### ISSUE TITLE
Create installer

### ISSUE THREAD

**Issue by Cselt** [Cselt/rf2-better-ui#2]
Create an installer so the users don't need to bother copy pasting the jar file

**Comment by Cselt** [comment]
You can find the rf2 install location with the following C# code
```
try
{
    using (var hklm = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, RegistryView.Registry64))
    using (var key = hklm.OpenSubKey(@"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Steam App 365960", false))
    {
        string rfactorloc = "" + key.GetValue("InstallLocation");

        if (!Directory.Exists(Path.Combine(rfactorloc, @"Installed\Vehicles")))
        {
            throw new NullReferenceException();
        }

        return rfactorloc;
    }
}
catch (NullReferenceException)
{
}
```

**Comment by Cselt** [comment]
Fixed in version 2.3.1
